### PR TITLE
Update dev ui start workflow page

### PR DESCRIPTION
**Pull request by AjayJagan**
**Description:**
This PR updates the Serverless workflow quarkus Dev-UI guide to incorporate the latest changes.
 - includes the new screenshots
 - Adds the information about the start workflow based on the form page.

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category
- [ ] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to setup automatic cherry-pick to another branch ?
</summary>

The cherry-pick action allows to setup automatic cherry-pick from `main` to a specific branch.

To allow it, you will need to add the corresponding label with pattern: `backport-{RELEASE_BRANCH}`.  
For example, if a backport to branch `1.26.x` is needed, then the label should be `backport-1.26.x`.

Once the PR is merged, the action will retrieve the commit and cherry-pick it to the desired branch.

*NOTE: You can still add label after the merge and it should still be cherry-picked.*
</details>

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc [hunk 1]
```diff
@@ -23,28 +23,32 @@ To clear the applied filters, you can click the *Reset to default* button. Also,
 image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-workflow-definitions-filter.png[]
 
 [[con-trigger-cloud-event-page]]
-== Trigger cloud event page
+== Start New Workflow page
 
-The Trigger cloud event page is used to trigger the cloud events related to a workflow definition. To navigate to the *Trigger cloud event* page, click on the play button in the *Actions* column of workflow definitions table.
+The Start New Workflow page is used start a workflow instance through a cloud event or a custom form depending on the workflow configuration. To navigate to the *Start New Workflow* page, click on the play button in the *Actions* column of workflow definitions table.
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc:28]:
```suggestion
The Start New Workflow page is used to start a workflow instance through a cloud event or a custom form, depending on the workflow configuration. To navigate to the *Start New Workflow* page, click on the play button in the *Actions* column of the workflow definitions table.
```

File: serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc [hunk 2]
```diff
@@ -23,28 +23,32 @@ To clear the applied filters, you can click the *Reset to default* button. Also,
...
 
-.Trigger cloud event page
+If there is no json schema for the workflow, then the workflow will be started by triggering a cloud event.
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc:30]:
```suggestion
*If there is no json schema for the workflow*, then the workflow will be started by triggering a cloud event.
```

File: serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc [hunk 3]
```diff
@@ -23,28 +23,32 @@ To clear the applied filters, you can click the *Reset to default* button. Also,
...
+To trigger a cloud event, you can use the *Cloud Event Type* and *Cloud Event Data* input fields to set the type of a cloud event and event payload in JSON format respectively, and click the *Start* button to start a workflow. The *Reset* button on the page resets the values entered in the given fields.
+
+.Starting a workflow using cloud event
 image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-trigger-cloud-events.png[]
 
-The *Trigger cloud event* page consists of the two fields, including *Type* and *Data*. To trigger a cloud event, you can use the *Type* and *Data* input fields to set the type of a cloud event and event payload in JSON format respectively, and click the *Send* button to start a workflow. The *Reset* button on the page resets the values entered in the given fields.
+If the workflow has a json schema configured in the `dataInputSchema` property, then a form is displayed in-order to start a new workflow instance. You can fill-in the required form details and click on the *Start* button to trigger the workflow.
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc:36]:
```suggestion
*If there is a json schema for the workflow* configured in the `dataInputSchema` property, then a form is displayed in-order to start a new workflow instance. You can fill-in the required form details and click on the *Start* button to trigger the workflow.
```

File: serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc [hunk 4]
```diff
@@ -23,28 +23,32 @@ To clear the applied filters, you can click the *Reset to default* button. Also,
...
+*If there is no json schema for the workflow*, then the workflow will be started by triggering a cloud event.
+To trigger a cloud event, you can use the *Cloud Event Type* and *Cloud Event Data* input fields to set the type of a cloud event and event payload in JSON format respectively, and click the *Start* button to start a workflow. The *Reset* button on the page resets the values entered in the given fields.
+
+.Starting a workflow using cloud event
 image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-trigger-cloud-events.png[]
 
-The *Trigger cloud event* page consists of the two fields, including *Type* and *Data*. To trigger a cloud event, you can use the *Type* and *Data* input fields to set the type of a cloud event and event payload in JSON format respectively, and click the *Send* button to start a workflow. The *Reset* button on the page resets the values entered in the given fields.
+*If there is a json schema for the workflow* configured in the `dataInputSchema` property, then a form is displayed in-order to start a new workflow instance. You can fill-in the required form details and click on the *Start* button to trigger the workflow.
```
Comment (paulovmr) [inline serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc:36]:
```suggestion
*If there is a json schema for the workflow* configured in the `dataInputSchema` property, then a form is displayed in order to start a new workflow instance. You can fill in the required form details and click on the *Start* button to trigger the workflow.
```

**Review by paulovmr — CHANGES_REQUESTED** [review]

**Review by paulovmr — APPROVED** [review]

**Review by pefernan — APPROVED** [review]

**Comment by ricardozanini** [comment]
@kaldesai can you take a look, please?

File: serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc [hunk 5]
```diff
@@ -23,28 +23,32 @@ To clear the applied filters, you can click the *Reset to default* button. Also,
 image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-workflow-definitions-filter.png[]
 
 [[con-trigger-cloud-event-page]]
-== Trigger cloud event page
+== Start New Workflow page
 
-The Trigger cloud event page is used to trigger the cloud events related to a workflow definition. To navigate to the *Trigger cloud event* page, click on the play button in the *Actions* column of workflow definitions table.
+The Start New Workflow page is used to start a workflow instance through a cloud event or a custom form, depending on the workflow configuration. To navigate to the *Start New Workflow* page, click on the play button in the *Actions* column of the workflow definitions table.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc:28]:
```suggestion
The *Start New Workflow* page is used to start a workflow instance through a cloud event or a custom form, depending on the workflow configuration. To navigate to the *Start New Workflow* page, click the play button in the *Actions* column of the workflow definitions table.
```

File: serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc [hunk 6]
```diff
@@ -23,28 +23,32 @@ To clear the applied filters, you can click the *Reset to default* button. Also,
...
 
-.Trigger cloud event page
+*If there is no json schema for the workflow*, then the workflow will be started by triggering a cloud event.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc:30]:
```suggestion
*If there is no JSON schema for the workflow, then the workflow is started by triggering a cloud event.
```

File: serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc [hunk 7]
```diff
@@ -23,28 +23,32 @@ To clear the applied filters, you can click the *Reset to default* button. Also,
...
+To trigger a cloud event, you can use the *Cloud Event Type* and *Cloud Event Data* input fields to set the type of a cloud event and event payload in JSON format respectively, and click the *Start* button to start a workflow. The *Reset* button on the page resets the values entered in the given fields.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc:31]:
```suggestion
To trigger a cloud event, you can use the *Cloud Event Type* and *Cloud Event Data* input fields to set the type of a cloud event and event payload in JSON format respectively and click the *Start* button to start a workflow. The *Reset* button on the page resets the values entered in the given fields.
```

File: serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc [hunk 8]
```diff
@@ -23,28 +23,32 @@ To clear the applied filters, you can click the *Reset to default* button. Also,
...
+
+.Starting a workflow using cloud event
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc:33]:
```suggestion
.Starting a workflow using a cloud event
```

File: serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc [hunk 9]
```diff
@@ -23,28 +23,32 @@ To clear the applied filters, you can click the *Reset to default* button. Also,
...
 image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-trigger-cloud-events.png[]
 
-The *Trigger cloud event* page consists of the two fields, including *Type* and *Data*. To trigger a cloud event, you can use the *Type* and *Data* input fields to set the type of a cloud event and event payload in JSON format respectively, and click the *Send* button to start a workflow. The *Reset* button on the page resets the values entered in the given fields.
+*If there is a json schema for the workflow* configured in the `dataInputSchema` property, then a form is displayed in order to start a new workflow instance. You can fill in the required form details and click on the *Start* button to trigger the workflow.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc:36]:
```suggestion
*If there is a JSON schema for the workflow configured in the `dataInputSchema` property, then a form is displayed to start a new workflow instance. You can fill in the required form details and click the *Start* button to trigger the workflow.
```

File: serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc [hunk 10]
```diff
@@ -23,28 +23,32 @@ To clear the applied filters, you can click the *Reset to default* button. Also,
...
+The *Reset* button is used to clear the form data.
 
-.Example values to trigger a workflow
-image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-trigger-cloud-event-values.png[]
+.Starting a workflow using form
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc:39]:
```suggestion
.Starting a workflow using the form
```

File: serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc [hunk 11]
```diff
@@ -23,28 +23,32 @@ To clear the applied filters, you can click the *Reset to default* button. Also,
...
+image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-trigger-cloud-events-custom-form.png[]
 
 You can also use the *Business key* text box to define a custom business key value to the workflow instance. If the *Business Key* field is blank, then an auto-generated business key is defined to the workflow instance.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc:42]:
```suggestion
You can also use the *Business key* text box to define a custom business key value to the workflow instance. If the *Business Key* field is blank, then an auto-generated business key is defined for the workflow instance.
```

**Review by kaldesai — APPROVED** [review]
@AjayJagan Added my minor suggestion. Please reach out to me if you have any queries related to my suggestions. Thank you!

**Comment by AjayJagan** [comment]
@kaldesai  thanks for the review. I have updated the PR with your suggestions. Let me know if I need to change anything else . :)

**Comment by kaldesai** [comment]
> @kaldesai thanks for the review. I have updated the PR with your suggestions. Let me know if I need to change anything else . :)

@AjayJagan I think that's it from my side. :) Thank you!

**Comment by AjayJagan** [comment]
> It seems this is ready to be merged? And probably we want to backport to 1.30.x branch so it will be included in DP3? @AjayJagan

@krisv  yes I have resolved all the comments and is ready to be merged and it should be backported to 1.30.x .

**Review by MarianMacik — APPROVED** [review]

**Pull request merged**